help-woke.rst

```diff
@@ -0,0 +1,46 @@
+===================================
+Inclusive language check exemptions
+===================================
+
+This page provides an overview of two inclusive language check exemption
+methods for files written in reST format. See the `woke documentation`_ for
+full coverage.
+
+Exempt a word
+-------------
+
+To exempt an individual word, place a comment on a line immediately preceding
+the line containing the word in question. This special comment must include the
+syntax ``wokeignore:rule=<SOME_WORD>``. For instance:
+
+.. code-block:: none
+
+   .. wokeignore:rule=whitelist
+   This is your text. The word in question is here: whitelist. More text.
+
+Here is an example of an exemption that acts upon an element (the string
+wokeignore:rule=master
```

Comment:
This won't work - it'll leave a `wokeignore:rule=master` both in the GitHub rendering and in the HTML output generated from RST.
I just tested this again... RST support for wokeignore is horrible. It doesn't work as described in line 18.
Let me figure something out ...